src/app.ts

```diff
@@ -12,15 +12,31 @@ interface TemperatureSummary {
   low: number
   average: number
 }
+let data: TemperatureReading[] = []
 
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  data = [...readings]
```

Comment:
could you explain to me why you have decided to spread an array into another array?